ckanext/ontario_theme/schemas/external/ontario_theme_dataset.json

```
@@ -882,5 +883,27 @@
       "validators": "ignore_missing unicode package_version_validator",
       "classes": ["form-group","col-md-12"]
     }
+    ,
+    {
+      "field_name": "schema",
+      "label": "Schema",
+      "preset": "resource_schema"
```

```suggestion
      "preset": "hidden_in_form"
```
If you use the "hidden_in_form" preset, you can remove `resource_schema.html` altogether and achieve the same results